src/app/qgisapp.cpp

```diff
@@ -8417,6 +8417,59 @@ void QgisApp::saveStyleFile( QgsMapLayer *layer )
   }
 }
 
+void QgisApp::loadStyleFile( QgsMapLayer *layer )
```

Comment:
It seems most of the code of this new method and QgsMapLayerStyleManagerWidget::loadStyle() is essentially identical, unless I'm missing something.
Perhaps it should be unified in a single utility method in a src/gui/qgsmaplayerstyleutils.h/cpp file ?
The same would hold for QgisApp::saveStyleFile( QgsMapLayer *layer ) vs QgsMapLayerStyleManagerWidget::saveStyle()